core/src/main/scala/org/apache/spark/SparkException.scala

```diff
@@ -269,48 +305,39 @@ private[spark] class SparkFileNotFoundException(
 private[spark] class SparkNumberFormatException(
     errorClass: String,
     errorSubClass: Option[String] = None,
-    messageParameters: Array[String],
+    messageParameters: Map[String, String],
```

Comment:
The exception SparkNumberFormatException is used only once by the error class CAST_INVALID_INPUT which is tested by *.sql.